### DISCUSSION TITLE
Version of zigbee-herdsman-converters package

### DISCUSSION THREAD

**Discussion by HB25** [Koenkk/zigbee2mqtt#10154] [Devices]
Hi all,

how can I figure out which version of zigbee-herdsman-converters is actually used? I ask because a device file does not contain the expected device.

Running npm info gives me this:

```
zigbee-herdsman-converters@14.0.344 | MIT | deps: 5 | versions: 698
Collection of device converters to be used with zigbee-herdsman
https://github.com/Koenkk/zigbee-herdsman-converters

keywords: xiaomi, tradfri, hue, wemo, zigbee, zigbee-shepherd

dist
.tarball: https://registry.npmjs.org/zigbee-herdsman-converters/-/zigbee-herdsman-converters-14.0.344.tgz
.shasum: fa07fc5d420b7618c37906cd5bd4a4c05bc7f6ee
.integrity: sha512-CTrRZBdOoaL4LWabcfQBct9dJd4dnYMcsNMsEJ2JQ+R20+1z+OIvKpECDLPDHQwIvTUPyxsObXtRfOM0exKu1g==
.unpackedSize: 2.7 MB

dependencies:
axios: ^0.24.0             buffer-crc32: ^0.2.13      https-proxy-agent: ^5.0.0  tar-stream: ^2.2.0         zigbee-herdsman: ^0.13.177 

```

14.0.344 contains the desired commit [#3481](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3481) but the local file philips.js in the modules directory does not.

I don't know much about node.js/npm. I guess I do something wrong but I haven't found a clue so far.

Does anybody have some hints? 

-Heiko

**Comment by darookee** [comment]
I have a similar problem and I can't figure out what's going on. I'm using the latest zigbee2mqtt docker image (1.22.1), but also tried `latest-dev`.

In [Version 14.0.347](https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/devices/lidl.js#L643), which is the one `npm info zigbee-herdsman-converters` gives me as beeing used (I think...?) there should be an entry for the device `TS0101`, but it is not in `devices.js`. On the other hand, there also is no `devices.js` in the repository but I can't figure out how it is built... I guess nodejs is weird in some ways... 🤷🏻

Trying to solve this I added external converts (using the same configuration as I expected in the `device.js`) and they work, but are not added in HomeAssistant... :-|
